### PR TITLE
Coverall changes

### PR DESCRIPTION
**Pull request by cjo20**
Don't merge, trying to make coverall happy.

### CODE REVIEW

**Comment by scottleibrand** [comment]
Is this ready to merge now?

**Comment by cjo20** [comment]
I was going to do more work on it in the future. If you want the tests added to this release and are happy with them, feel free to merge.

File: tests/profile.test.js [hunk 1]
```diff
@@ -14,14 +14,16 @@ describe('Profile', function ( ) {
...
+                { offset: 480, high: 115, low: 95}
             ]
         }
         , temptargets: [
         ]
         , isf: {
             sensitivities: [
-                { offset: 0, i: 0, x: 0, start: '00:00:00', sensitivity: 100 }
+                { offset: 0, i: 0, x: 0, start: '00:00:00', sensitivity: 100 },
+                { offset: 480, i: 16, x: 1, start: '08:00:00', sensitivity: 80 }
```
Comment (scottleibrand) [inline tests/profile.test.js]: This is breaking tests between 8am and midnight.

File: tests/profile.test.js [hunk 2]
```diff
@@ -55,4 +57,74 @@ describe('Profile', function ( ) {
 
     });
 
+
+    it('should should honour override_high_target_with_low', function () {
+        var profile = require('../lib/profile')(_.merge({}, baseInputs, {override_high_target_with_low: true}));
+        profile.max_iob.should.equal(0);
+        profile.dia.should.equal(3);
+        profile.sens.should.equal(100);
```
Comment (scottleibrand) [inline tests/profile.test.js]: This is broken between 8am and midnight.

**Pull request merged**